src/control/cmd/ddb/main.go

```diff
@@ -122,7 +122,33 @@ a file in the VOS format. It offers both a command line and interactive
 shell mode. If neither a single command or '-f' option is provided, then
 the tool will run in interactive mode. In order to modify the VOS file,
 the '-w' option must be included. If supplied, the VOS file supplied in
-the first positional parameter will be opened before commands are executed.`
+the first positional parameter will be opened before commands are executed.
+
+Many of the commands take a vos tree path. The format for this path
+is [cont]/[obj]/[dkey]/[akey]/[extent].
```

Comment:
Just wondering how a user would know these paths. I think this is fine for the time being, but I'm curious about the use cases for the tool.

Reply:
One of the commands is "ls" which lists the paths. It would be used to determine which paths the other commands might want to be run on.